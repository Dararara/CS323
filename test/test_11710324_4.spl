int test_1_m01(int a, int b)
{
  int code, new;
  code = 0xf0;
  new = 0xzz;
  new = $ + new - 0xkkk;

  return hello('a, new);
}