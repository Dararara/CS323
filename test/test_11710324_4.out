Error type A at Line 5: unknown lexeme 0xzz
Error type A at Line 6: unknown lexeme $
Error type A at Line 6: unknown lexeme 0xkkk
Error type B at Line 8: Missing closing '
